api-test/run-all.sh

```
@@ -0,0 +1,108 @@
+#     SUITE_WAIT_SECONDS    seconds to poll the conformance suite's
+#                           /api/runner/available before the conformance surface
+#                           runs (default: 90; 0 disables the wait)
+#
+# Example:
+#   ESIGNET_BASE_URL=https://esignet-thunder1.esdev.mosip.net/v1/esignet \
+#   KEYCLOAK_TOKEN_URL=https://iam.esdev.mosip.net/auth/realms/mosip/protocol/openid-connect/token \
+#   KEYCLOAK_CLIENT_ID=mosip-pms-client KEYCLOAK_CLIENT_SECRET=*** \
+#   ./run-all.sh
+#
+set -uo pipefail
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)